xknx/knx/dpt_float.py

```diff
@@ -153,6 +156,7 @@ class DPTTemperature(DPT2ByteFloat):
     value_min = -273
     value_max = 670760
     unit = "°C"
+    ha_device_class = "temperature"
```

Comment:
here the home assistant device_class strings are assigned  for corresponding DPT  #194